searx/webapp.py

```diff
@@ -1113,18 +1174,18 @@ def stats():
     engine_stats = get_engines_stats(filtered_engines)
     engine_reliabilities = get_reliabilities(filtered_engines, checker_results)
 
-    SORT_PARAMETERS = {
+    sort_parameters = {
```

Comment:
It should be at the top of the file actually, declared as "constant"

Reply:
good point, I will move it to the top of the file

Reply:
fixed here https://github.com/searxng/searxng/commit/e295bbf0679fe1def6983be969b735ef20d26c19#diff-ab9553a5df12bab91ccac7a4b9af78fbd160e35bffae9a340bf75ed736d50a07R164-R171